.github/workflows/run-tests.yml

```diff
@@ -19,6 +19,4 @@ jobs:
         uses: actions/setup-node@v3
       - run: npm ci
       - run: npm run build
-      - run: cd test-site
-      - run: npm i
-      - run: npm run build
+      - run: cd test-site && npm i && npm run build
```

Comment:
could this job use the reusable run-tests workflow and pass this last step in as the test script?